x/authority/migrations/v3/migrate.go

```diff
@@ -0,0 +1,21 @@
+package v3
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	"github.com/zeta-chain/node/x/authority/types"
+)
+
+type authorityKeeper interface {
+	SetAuthorizationList(ctx sdk.Context, list types.AuthorizationList)
+}
+
+// MigrateStore migrates the authority module state from the consensus version 2 to 3
+func MigrateStore(
+	ctx sdk.Context,
+	keeper authorityKeeper,
+) error {
+	// It is okay to not validate here, as the authorization list is fixed and will not change
+	keeper.SetAuthorizationList(ctx, types.DefaultAuthorizationsList())
```

Comment:
Not a good approach because we might already have authorization different from default, why ntot using addAuthorization?

Reply:
I was considering having the DefaultAuthorizationsList mapped to a document we maintain , so that we are always confident on what the authorization list is. 
I can still modify the logic to adding if that seems better

Reply:
I think it's proper using AddAuthorization

Reply:
@kingpinXD seems we're still overwriting the list